site/en/docs/privacy-sandbox/topics/index.md

```diff
@@ -2,328 +2,70 @@
-[FLEDGE](/docs/privacy-sandbox/fledge) and [Attribution Reporting](/docs/privacy-sandbox/attribution-reporting/) APIs.
-
-To take part, [register for an origin trial token](/origintrials/#/view_trial/771241436187197441).
-
-Once you have successfully enrolled in the trial, you can try out the Topics JavaScript API on pages
-that provide a valid trial token:
-
-*   As a meta tag in the &lt;head&gt;:<br>
-
-    `<meta http-equiv="origin-trial" content="TOKEN_GOES_HERE">`
-
-*   As an HTTP header:<br>
-
-    `Origin-Trial: TOKEN_GOES_HERE`
+## Try the Topics API
 
-*   By providing a token programmatically:<br>
+Topics is not currently available by default in any version of Chrome, but you can activate the API in two ways, as a single user or at scale:
 
-    ```javascript
-    const otMeta = document.createElement('meta');
-    otMeta.httpEquiv = 'origin-trial';
-    otMeta.content = 'TOKEN_GOES_HERE';
-    document.head.append(otMeta);
-    ```
+-   The Topics API demo allows you to try it out as a single user.
+-   The Topics origin trial allows you to try the API at scale with your website users.
 
-An iframe running Topics code&mdash;such as a `document.browsingTopics()` call to observe
-topics&mdash;will need to provide a token that matches its origin.
+### Try the demo
 
-{% Aside 'caution' %}
-
-Not all users are eligible for the Privacy Sandbox Relevance and Measurement
-origin trial, even on pages that provide a valid trial token.
-
-[Testing the Privacy Sandbox ads relevance and measurement APIs](/blog/privacy-sandbox-unified-origin-trial#eligible-users)
-explains why this is, and shows how you can (and should) detect if an origin trial feature is
-available before attempting to use it.
-
-{% endAside %}
-
-
-## Test with `chrome://flags` or feature flags {: #feature-flags}
-
-There are two ways to try the Topics API as a single user, running Chrome 101 or above:
-
-*  Enable `chrome://flags/#privacy-sandbox-ads-apis`
-*  Run Chrome from the command line with the following flags:
-
-``` text
---enable-features=BrowsingTopics,PrivacySandboxAdsAPIsOverride,OverridePrivacySandboxSettingsLocalTesting
-```
+The demo of the Topics API is at [topics-demo.glitch.me](https://topics-demo.glitch.me/). It explains how to try out and debug the API for a single user.
 
-The [Topics demo](#demo) shows how to use additional flags to adjust settings such as epoch
-length. If you access the Topics API by running Chrome with command-line flags, don't
-set `chrome://flags`, as these can override command-line settings.
+You can also run the Topics [colab](/docs/privacy-sandbox/topics/colab/) to try out the Topics [classifier model](/docs/privacy-sandbox/topics/topic-classification/#classifier-model).
 
-[Run Chromium with flags](https://www.chromium.org/developers/how-tos/run-chromium-with-flags)
-explains how to set flags when running Chrome and other Chromium-based browsers from the command
-line.
+### Take part in a Topics origin trial
```

Comment:
```suggestion
### Test Topics in an origin trial
```

This may clarify that one can't find a "Topics origin trial" but rather has to find the relevence and measurement origin trial to test Topics.